### ISSUE TITLE
Notebooks

### ISSUE THREAD

**Issue by anschaible** [ufuk-cakir/rubix#95]
- Clean up the current existing notebooks
- Create explanation notebooks to use later in the documentation

**Comment by ufuk-cakir** [comment]
> that I have ti set a SPS_HOME environment variable

I think this is not related to the config but rather the package @TobiBu setup to load different SSP models, if I remember correcty

**Comment by TobiBu** [comment]
yeah, for fsps to work properly, the [pyfsps docs](https://dfm.io/python-fsps/current/installation/) mention to set the environment variables for pyfsps at installation time.

we could add a note to follow the pyfsps installation to make this notebook work.

**Comment by TobiBu** [comment]
added a small info to the notebook. Now we should be ready to merge, right?
